website/addons/twofactor/views.py

```diff
@@ -20,7 +21,8 @@ def user_settings(user_addon, *args, **kwargs):
     if user_addon.verify_code(code):
         user_addon.is_confirmed = True
         user_addon.save()
-        return {'message': 'Successfully verified two-factor authentication.'}, http.OK
+        push_status_message('Successfully verified two-factor authentication.', kind='success')
+        return http.OK
```

Comment:
This is incorrect. Views must return either response data or a tuple where the first element is the response data, 2nd is the status code (optional), 3rd is headers (optional).